columnar/src/column_index/optional_index/mod.rs

```diff
@@ -81,22 +81,22 @@ impl BlockVariant {
 #[derive(Clone)]
 pub struct OptionalIndex {
     num_rows: RowId,
-    num_non_null_rows: RowId,
+    num_non_null_docs: RowId,
     block_data: OwnedBytes,
     block_metas: Arc<[BlockMeta]>,
 }
 
 impl Iterable<u32> for &OptionalIndex {
     fn boxed_iter(&self) -> Box<dyn Iterator<Item = u32> + '_> {
-        Box::new(self.iter_rows())
+        Box::new(self.iter_docs())
```

Comment:
@PSeitz The reason why columnar was originally not using "docs" but "rows" is because this is a columnar format. The concept of docs is specific to search engine or document DBs.

I am not sure this makes the code any better, but ok.

Reply:
It's been some time, but we decided to use docs everywhere. (I can make a full PR later)
The naming is confusing to me, as the meaning of row is different to the meaning in the DB I worked before. The row docs naming also caused a bug before (or 2? not sure).